package.json

```diff
@@ -37,13 +37,14 @@
     "down": "docker-compose down",
     "wp": "docker-compose exec -u www-data wordpress wp",
     "install-if-deps-outdated": "node bin/install-if-deps-outdated.js",
+    "typecheck": "tsc --noEmit",
     "lint": "npm run lint:js && npm run lint:css && npm run lint:php",
     "lint:css": "stylelint 'client/**/*.scss' 'client/**/*.css' 'assets/**/*.scss' 'assets/**/*.css'",
-    "lint:js": "eslint . --ext=js,jsx",
+    "lint:js": "npm run typecheck && eslint . --ext=js,jsx,ts,tsx",
```

Comment:
Let's just inline typecheck command and use `tsc --noEmit` instead.

Reply:
Fixed in 7879887f27998150361323ff948805177223c0f0